orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/servergroup/ServerGroupCacheForceRefreshTask.groovy

```diff
@@ -138,10 +143,17 @@ class ServerGroupCacheForceRefreshTask extends AbstractCloudProviderAwareTask im
 
     boolean finishedProcessing = true
     stageData.deployServerGroups.each { String region, Set<String> serverGroups ->
-      finishedProcessing = finishedProcessing && serverGroups.every { String serverGroup ->
```

Comment:
Uncovered a scenario involving smaller accounts that indexed quickly (< 30s cycles) and operations affecting multiple server groups.

It was not 100% reliable to expect that every single server group would be processed in the same `CloudDriver.loadData()` cycle. 

In reality we should check all server groups at once and avoid checking the same server group over and over again.